lib/rust/frp/src/nodes.rs

```diff
@@ -1793,8 +1793,8 @@ impl<T: EventOutput> OwnedTrace<T> {
 
 impl<T: EventOutput> stream::EventConsumer<Output<T>> for OwnedTrace<T> {
     fn on_event(&self, stack: CallStack, event: &Output<T>) {
-        debug!("[FRP] {}: {:?}", self.label(), event);
-        debug!("[FRP] {}", stack);
+        warn!("[FRP] {}: {:?}", self.label(), event);
+        warn!("[FRP] {}", stack);
```

Comment:
Make it `debug!` again.

Reply:
As agreed, lets leave it as warn to make debugging fro easier now.